.github/workflows/pr.yml

```diff
@@ -83,6 +83,10 @@ jobs:
             git --no-pager diff
             exit 1
           fi
+      - uses: dominikh/staticcheck-action@v1.1.0
```

Comment:
Not something that needs to be addressed in this PR, but wondering: would it be better if this was its own PR check?

Reply:
TBH, this could apply to other things in this PR check workflow as well

Reply:
yeah we could clean up the workflow a bit